apps/spruce/src/pages/waterfall/InactiveVersions/index.tsx

```diff
@@ -25,10 +28,19 @@ export const InactiveVersionsButton: React.FC<Props> = ({
       0,
     ) ?? 0;
   const [modalOpen, setModalOpen] = useState(false);
+  const [revisionFilter] = useQueryParam<string | null>(
+    WaterfallFilterOptions.Revision,
+    null,
+  );
```

Comment:
Is there any reason not to type this consistently and use an empty string instead of null? Seems like it may be easier to only have to handle one non-value, but mostly just curious.

Reply:
I mainly did it to ensure the usage is consistent throughout the app. If the value is null in one place I thought it should be nullable everywhere else since its the same origin. I don't have a strong preference either way.